### PR TITLE
Remove IL2050 in System.Management

### PR DESCRIPTION
**Pull request by kant2002**
Remove unused functions which trigger IL2050
As per discussed in #54317

### CODE REVIEW

File: src/libraries/Common/src/Interop/Windows/Ole32/Interop.CreateStreamOnHGlobal.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Ole32
     {
-        [DllImport(Libraries.Ole32)]
-        internal static extern int CoGetObjectContext([MarshalAs(UnmanagedType.LPStruct)] Guid riid, out IntPtr ppv);
+        [DllImport(Libraries.Ole32, PreserveSig = false)]
+        internal static extern IStream CreateStreamOnHGlobal(IntPtr hGlobal, bool fDeleteOnRelease);
```
Comment (stephentoub) [inline src/libraries/Common/src/Interop/Windows/Ole32/Interop.CreateStreamOnHGlobal.cs:12]: Took me a sec to figure out what was going on here.  Thanks for fixing the swap.

**Comment by stephentoub** [comment]
cc: @eerhardt

File: src/libraries/Common/src/Interop/Windows/Ole32/Interop.CreateStreamOnHGlobal.cs (hunk 1 above)
Comment (kant2002) [reply src/libraries/Common/src/Interop/Windows/Ole32/Interop.CreateStreamOnHGlobal.cs:12]: I hear you. I was in similar position before :)

**Review by eerhardt — APPROVED** [review]
This LGTM. Thanks for the fix up here.

**Pull request merged**